<a href="https://colab.research.google.com/github/goulartsursula/python-uerj/blob/master/Exercicios/Ursula_Goulart_Aula_1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

**Questão 01:** Se fizermos uma corrida de 10 quilômetros em 43 minutos e 30 segundos, qual será o seu tempo médio por milha? Qual é a sua velocidade média em milhas por hora?

In [ ]:
43 * 60

2580

10 quilômetros em 2610 segundos

In [ ]:
(1.61 * 2610) / 10

420.21000000000004

Tempo médio por milha é de 420.21 segundos, aproximadamente 7 minutos.

In [ ]:
10 / 1.61

6.211180124223602

10km = 6.21 mi

In [ ]:
(2610 / 3600)

0.725

43 minutos e 30 segundos é equivalente a 0.725 horas.



In [ ]:
6.21 / 0.725

8.565517241379311

Velocidade média de 8.566 mi/h

**Questão 02:** Desde sua varanda você escuta o som do primeiro fogo artificial do reveillon 3 segundos depois de ver a luz, qual a distância? (o som tem velocidade 343 m/s e a luz ($3\times 10^8$ m/s).

In [ ]:
v = 363
t = float(input('Depois de quanto tempo que viu a luz ouviu o som do fogo artificial?'))
print ('A distância entre a varanda e o fogo artificial é {}m'.format(t*t))

**Questão 03:** Ache os zeros da função
 y = 3 *x^2 - 4 *x -10

In [ ]:
import math

print('Uma função do segunda grau tem a forma y = ax²+ bx +c, sendo c diferente de zero')
a = int(input('Insira o valor de a = '))
b = int(input('Insira o valor de b = '))
c = int(input('Insira o valor de c = '))
print('A função é: y = {}x² + {}x + {}'.format(a, b, c))
delta = b**2 - 4*a*c
raiz = math.sqrt(delta)
y1 = ((-1)*b)/(2*a) - ((raiz)/(2*a))
y2 = ((-1)*b)/(2*a) + ((raiz)/(2*a))
print ('A raízes são {:.2f} e {:.2f}'.format(y1, y2))

**Questão 04:** Se, ao meiodia, a sombra de um poste de 5 m de altura tem apenas 50 cm de comprimento no chão, qual o ângulo zenital do sol?

In [ ]:
import math

print('O objetivo é calcular o ângulo zenital do sol, para isso é necessário utilizar algo como referência.')
altura = float(input('Qual a altura desse objeto?'))
comp_sombra = float(input('Qual o comprimento da sombra desde objeto?'))
angulo = altura / comp_sombra
print('O ângulo zenital do sol é de {:.2f} radianos'.format(math.atan(angulo)))

**Questão 05:** Calcule o seu $IMC = \dfrac{M}{A^2}$ (com a massa em Kg e a altura em metros). Um valor saudável estara --em geral-- entre 20-25. Um bebê de 6 meses "gorducho" tem 70 cm de "comprimento" e 11 kg de massa, qual o IMC dele?

Informações retiradas do endereço eletrônico da Sociedade Brasileira de Endocrinologia e Metabologia.

In [ ]:
print('Cálculo do índice de massa corporal.')
altura = float(input('Qual a sua altura? Dê a sua altura em metros.'))
peso = float(input('Quanto você pesa? Dê o seu peso em quilogramas.'))
imc = peso/(altura**2)
if imc<=18.5:
  print('Seu IMC é de {:.0f}. Baixo peso. Grau de obesidade 0'.format(imc))
elif imc>18.5 and imc<=24.9:
  print('Seu IMC é de {:.0f}. Peso normal. Grau de obesidade 0.'.format(imc))
elif imc>=25 and imc<=29.9:
  print('Seu IMC é de {:.0f}. Sobrepeso. Grau de obesidade I.'.format(imc))
elif imc>=30 and imc <=39.9:
  print('Seu IMC é de {:.0f}. Obesidade. Grau de obesidade II.'.format(imc))
elif imc>40:
  print('Seu IMC é de {:.0f}. Obesidade grave. Grau de obesidade III.'.format(imc))

**Questão 06:** Calcule a velocidade final de um objeto em queda livre a partir de 3 metros de altura (sem resistencia do ar). Calcule o tempo que esse objeto demora para cair.

In [ ]:
import math

altura = float(input('A que altura está o objeto, em metros?'))
velocidade = math.sqrt(2*9.8*altura)
tempo = velocidade/9.8
print('O objeto atinge uma velocidade final de {} m/s² e chega ao chão em {} segundos.'.format(round(velocidade,2), round(tempo,2)))